samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+A sample implementation of a power manager app that uses the Zephyr
```

Comment:
Make this a sentence by changing the start:

     This sample demonstrates a power manager app ...